ZFSin/zfs/module/zfs/zfs_vnops_windows_lib.c

```diff
@@ -246,14 +246,46 @@ NTSTATUS mountmgr_add_drive_letter(PDEVICE_OBJECT mountmgr, PUNICODE_STRING devp
 	return Status;
 }
 
-NTSTATUS mountmgr_get_drive_letter(PDEVICE_OBJECT mountmgr, PUNICODE_STRING devpath, char *savename)
+/*
+* check if valid mountpoint, like \DosDevices\X:
+*/
+BOOLEAN MOUNTMGR_IS_DRIVE_LETTER_A(char *mountpoint) {
+	UNICODE_STRING wc_mpt;
+	wchar_t buf[PATH_MAX];
+	mbstowcs(buf, mountpoint, 200);
```

Comment:
nit: `200` - perhaps sizeof(buf) or sizeof(wchar_t)*PATH_MAX ?

Reply:
that's probably some copy-paste leftover... sizeof(buf) will do it, i guess :)